codecov.yml

```diff
@@ -4,64 +4,9 @@ coverage:
     patch: off
     project:
       default:
-        target: 60%
```

Comment:
Why did we remove all of these targets?

Reply:
TLDR: these targets are useless and are impossible to update properly (I set them initially, my bad!)

Since a bunch of tests were moved from 'unit' to 'integration, coverage got lowered a bunch in various packages.
I initially tried to adjust these % and packages position in each % target, pulled my hair off, and realized that each % target listed in codecov.yml are the average % coverage for all the packages listed in them (I thought it was a % target for each package, not their average). And I am still not too sure how they calculate this 'average' (go test doesn't offer such thing either).

One solution would be to have a coverage target per package, but, visually speaking, that would make our workflows list in our CI become mega long and I'd personally avoid.